lib/langs/python.rb

```diff
@@ -13,10 +17,8 @@ class PythonLang
 
   class << self
 
-    include LanguageHelpers
-
     # Defaults for projects in this language
-    def locations(name)
+    def locations(name = nil)
       ['requirements.txt']
```

Comment:
Why only requirements.txt here and not pipfile?

Reply:
Yeah, good question. I had assumed that the most common use case is still `requirements.txt`, as Pipfiles are pretty new. And unlike `Gemfile` and `gemspec`, a python project will use only one of them. But we might as well check for both of them by default!

Reply:
Added `Pipfile` now. This means, however, that we'll always be *looking* for both -- and there will, in practice, never *be* both. The risk is thus that either one of them will show up as 'missing' on the info view. Not sure what's best here.

Reply:
Look for one or the other, somehow? 😬

Reply:
The user can always set `locations: [Pipfile]` or `locations: [requirements.txt]` in the config to check for only one. But I don't see a way of automagically checking whether a project is using only the one or the other, other than fetching all locations in the worker, and just calling `.compact` on the resulting Array?

Reply:
OK, fair enough. Look for both by default, unless set otherwise in the config. 👍